src/arc_spice/variational_pipelines/RTC_variational_pipeline.py

```diff
@@ -0,0 +1,545 @@
+import copy
+import logging
+from typing import Any, Union
+
+import torch
+from torch.nn.functional import softmax
+from transformers import (
+    AutoModelForSequenceClassification,
+    AutoTokenizer,
+    TranslationPipeline,
+    pipeline,
+)
+
+from arc_spice.variational_pipelines.dropout_utils import (
+    count_dropout,
+    dropout_off,
+    dropout_on,
+    set_dropout,
+)
+
+# From huggingface page with model:
```

Comment:
remove?

Reply:
Yeah, this was left over from old versions of this file. Removed.